### ISSUE TITLE
Apply QA for 2022 Content

### ISSUE THREAD

**Issue by c-schuler** [cqframework/opioid-cds-r4#340]

**Comment by alackerbauer** [comment]
Q&A report can be found here: http://build.fhir.org/ig/cqframework/opioid-cds-r4/qa.html

**Comment by alackerbauer** [comment]
cannot find value set that replaces    
```
 <li><a href="ValueSet-medicationrequest-category-community.html">MedicationRequest Category Community</a></li>
```

i can only find it in the archive. was this replaced by another one or removed entirely?

**Comment by alackerbauer** [comment]
I did a comparison of what is listed in `input/pagecontent/terminology.xml` and which value sets are present in `input/vocabulary/valueset`.

There is a discrepancy that I'd need feedback on before resolving the last 3 errors from the comment above:

This is what we have in the value set folder but not listed in the terminology.xml:

* Condition Encounter diagnosis category
* Synthetic opioid urine drug screening test
* Sickle cell diseasees
* Phencyclidine urine drug screening tests
* Opioid treatment assessment procedure
* Opiate specific urine drug screening tests
* Non-opioid cns depressant medications
* Methadone medications
* Methadone drug urine screening tests
* General opiate urine drug screening tests
* Conditions documenting sleep disordered breathing
* Cocaine urine drug screening
* Amphetamines class medications
* Amphetamine drug urine screening tests
* Alcohol drug urine screening tests

This is what is mentioned in the list in terminology.xml but cannot be found in the vocabulary folder:
*CDC malignant cancer conditions
* MedicationRequest Category Community
* MedicationRequest Status Active
* Action Indicators

**Comment by alackerbauer** [comment]
try this for value sets:   {% include list-simple-codesystems.xhtml %} check how it renders. 

if not, add all valuesets from the valueset folder to the list.

also, add everything that was checked but not resolved for now to supporess warnings file.

**Comment by alackerbauer** [comment]
the include list-simple-codesystems.xhtml works generally speaking. It uses the ValueSet.title for the hyperlink texts. But the ValueSet titles aren't as speaking as the manually listed ones. E.g.:

ValueSet title: `ValueSet - US Core Health Concern Condition Category`
Manually defined name (nowhere to be found in this ValueSet): `Condition Category US Core Health Concern`

**Comment by alackerbauer** [comment]
Tried to fix the QA warning for embedded HTML in valueMarkdown of a ValueSet's rulestest. Was not successful, neither escaping all special characters (e.g. to unicode), neither putting the code into a MD code block solved the problem. I worked with a JSON escaper: https://www.freeformatter.com/json-escape.html#before-output as well as manual unicode and markdown changes. 

If ok, I will ask in zulip since this is a warning that shouldn't happen.
